src/WCSimWCDAQMessenger.cc

```diff
@@ -46,6 +46,7 @@ WCSimWCDAQMessenger::WCSimWCDAQMessenger(WCSimEventAction* eventaction) :
   TriggerChoice->SetCandidates(
 			       "NDigits "
 			       "NDigits2 "
+			       "NoTrigger "
```

Comment:
Please remember to add the new trigger @ line 44 (SetGuidance) too.
Take a look at the pdf document in /doc - it gives a complete list of things to do when adding new triggers

Reply:
done